app/user/serializers.py

```diff
@@ -1,116 +1,72 @@
-    )
+    email = serializers.EmailField(required=True)
 
-    email = serializers.EmailField(
-        required=True,
-        validators=[UniqueValidator(queryset=User.objects.all())],
-    )
+    class Meta:
+        fields = ["email"]
 
-    password = serializers.CharField(
-        max_length=128,
-        min_length=8,
-        write_only=True,
-        validators=[
-            validate_password_digit,
-            validate_password_uppercase,
-            validate_password_symbol,
-            validate_password_lowercase,
-        ],
-    )
+    def validate(self, attrs: Any) -> Any:
 
-    class Meta:
-        model = User
-        fields = ("email", "username", "password")
-
-    @staticmethod
-    def send_email(user: Any, request: Any) -> None:
-
-        current_site_info = get_current_site(request)
-        email_body = render_to_string(
-            "email_verification.html",
-            {
-                "user": user,
-                "domain": current_site_info.domain,
-                "uid": urlsafe_base64_encode(force_bytes(user.pk)),
-                "token": account_activation_token.make_token(user),
-            },
-        )
-
-        send_mail(
-            "Verify  your email!",
-            email_body,
-            EMAIL_USER,
-            [user.email],
-            fail_silently=False,
-        )
-
-    def create(self, validated_data: Any) -> Any:
         request = self.context.get("request")
-        user = User.objects.create_user(**validated_data)
-        self.send_email(user, request)
+        email = attrs.get("email")
+        user = generate_reset_token(email)
+
+        if user:
+            email_data = create_reset_email(request, *user)
+            send_email("password_reset.html", email_data)
 
-        return user
+        return super().validate(attrs)
 
 
-class VerifyEmailSerializer(serializers.Serializer):
-    token = serializers.CharField()
-    uidb64 = serializers.CharField()
+class VerifyPasswordResetSerializer(serializers.Serializer):
+
+    password = serializers.CharField(
+        write_only=True,
+        min_length=6,
+    )
 
     class Meta:
-        fields = ("token", "uidb64")
+
+        fields = ["password"]
 
     def validate(self, data: Any) -> Any:
-        user = None
-        try:
-            user_id = force_str(urlsafe_base64_decode(data.get("uidb64")))
-            user = User.objects.get(pk=user_id)
 
-        except (TypeError, ValueError, OverflowError, User.DoesNotExist):
+        password = data.get("password")
+        url_kwargs = self.context.get("kwargs")
+        token = url_kwargs.get("token")  # type: ignore[union-attr]
+        encoded_pk = url_kwargs.get("encoded_pk")  # type: ignore[union-attr]
+
+        if token is None or encoded_pk is None:
             raise serializers.ValidationError(
-                "Invalid user id", code="invalid_code"
+                {"detail": "This field is required"}
```

Comment:
this error is not specific it does not say which one is required, and I do not think it is necessary (the if statement)